src/main/java/org/sagebionetworks/bridge/sdk/models/surveys/Unit.java

```diff
@@ -0,0 +1,35 @@
+package org.sagebionetworks.bridge.sdk.models.surveys;
+
+public enum Unit {
```

Comment:
Add comment to enums in SDK explaining that these serialize as lower-case text in JSON.